fiwareglancesync/app/app.py

```diff
@@ -119,6 +122,7 @@ def bad_method(error):
     }
     ''' % str(httplib.METHOD_NOT_ALLOWED)
 
+    logger_api.warn(msg)
```

Comment:
this error is METHOD_NOT_ALLOWED, usually POST, DELETE, etc.. bad selection of the verb